src/lib.rs

```diff
@@ -49,10 +49,13 @@ mod error;
 mod iter;
 pub mod parse;
 
-/// Reexports of extension traits.
-pub mod exts {
-    pub use super::display::DisplayHex;
-    pub use super::parse::FromHex;
+/// Reexports of the common crate types and traits.
+pub mod prelude {
+    #[doc(inline)]
+    pub use crate::{
+        display::DisplayHex,
+        parse::{FromHex, HexToArrayError, HexToBytesError},
+    };
```

Comment:
In 326aad38caede931157b7e5f9d6ba6b650319670:

At first I didn't like this, but I see that everything in the `exts` module is re-exported at the crate root. So it seems like `exts` was already supposed to be a sort of `prelude` module, and I agree it would be better if it were just named `prelude`.

Reply:
Kind of, it was meant to only contain extension traits, preludes often contain other things. But I guess it's better to use `prelude` anyway.